cmd/kes/server.go

```diff
@@ -139,13 +139,21 @@ func startServer(addrFlag, configFlag string) error {
 		return err
 	}
 
+	// Read the config file before looking up the
+	// local network interfaces. We may not know the
+	// server addr yet since a user may not specified
+	// one on the command line.
 	rawConfig, err := kesconf.ReadFile(configFlag)
 	if err != nil {
 		return err
 	}
 	if addrFlag == "" {
```

Comment:
We have to check `addrFlag` first since that has the highest precedence. So you're suggesting this?
```
if addrFlag == "" {
    if rawConfig.Addr == "" {
	addrFlag = "0.0.0.0:7373"
    } else {
        addrFlag = rawConfig.Addr
    }
}
```

Reply:
Used a switch for less indention